bin/main.rb

```diff
@@ -0,0 +1,92 @@
+# rubocop:disable Metrics/PerceivedComplexity, Metrics/CyclomaticComplexity
+
+# !/usr/bin/env ruby
+
+require '../lib/classes.rb'
+
+puts 'Welcome to our Tic-Tac-Toe Game!'
+
+puts 'Player 1, please type your beautiful name?'
+player_one = Player.new(gets.chomp, 'X')
+puts "Nice name #{player_one.name}!"
+puts 'Please, notice that, from now on, you are going to be represented by the "X" characters on the board, right?'
```

Comment:
- [ ] This was confusing. The idea to make the game more interactive is awesome. But I did not know what to type to continue. You could add: 'press enter to continue'